src/haskell/devcontainer-feature.json

```
@@ -0,0 +1,54 @@
+{
+    "id": "haskell",
+    "version": "0.0.1",
+    "name": "Glasgow Haskell Compiler",
+    "documentationURL": "https://github.com/brendandburns/ghc-feature",
+    "description": "Installs the Glasgow Haskell Compiler",
+    "options": {
+        "version": {
+            "type": "string",
+            "proposals": [
+                "latest",
+                "recommended",
+                "9.4.4",
+                "9.2.7"
+            ],
+            "default": "recommended",
+            "description": "Select or enter a GHC Version to install"
+        },
+        "cabal_version": {
```

```suggestion
        "cabalVersion": {
```

From https://containers.dev/implementors/features/#options-property, we use camel casing! Can we update this?

Note: The ID becomes the name of the environment variable in all caps

We'd need to change the `install.sh` to adapt it.
eg. CABAL_VERSION="${CABALVERSION:-"recommended"}"

---

done